### ISSUE TITLE
Seasonal gear showing up as both Class:None and Class:[correct class]

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitica#10727]
Per Il Mago, with a screenshot: https://snag.gy/u0XQ8I.jpg

> _Il Mago (c073342f-4a65-4a13-9ffd-9e7fa5410d6b): "Bug: the seasonal equipment for my own class appear in the Market under Class:none (for example, Tulip set from Spring 2018 appear both under Class:Mage and Class:none)
> 
> I don't know if it is all sets, but there are a lot"_

**Comment by kirsty-tortoise** [comment]
Had Il Mago by any chance already bought and then lost the seasonal equipment that was showing up on both?

That's a case I've been able to replicate this in (and seems to have nothing to do with seasonal gear). See this code:
https://github.com/HabitRPG/habitica/blob/bd81d27145b99842e386f0c0ffe793f03abf9ace/website/common/script/libs/shops.js#L190-L196

My understanding of Javascript is that order of precedence means that if `user.items.gear.owned[gear.key] === false` (which it does if you've purchased and then lost an item) then the entire statement will evaluate to true.

Basically any item that has been purchased and then lost will appear in Class:None. I can fix this with brackets if this is not how it is supposed to be.

**Comment by shanaqui** [comment]
Il Mago definitely previously had the items and I believe they were all removed by @alys to mimic how rebirth used to work.

**Comment by majestic-owl448** [comment]
Ow, yes, there is everything under Class:none, I had not noticed the class equipment!

And yes, any equipment that was in my account was set with a value of false, in the same way the Orb of Rebirth old edition did and the future "empty the Armoire" feature would do
